Contacts: Verifying contact ID causes verifyee's app to crash

Steps to reproduce:

User A and User B

1. User A sends contact request to user B
2. User B accepts contact request
3. User A sends ID verification to user B
4. User B responds to ID verification
5. User A's app crashes

@caybro also reported that he couldn't start the app anymore after that.

I can reproduce as well:

```
qrc:/imports/shared/status/StatusChatInput.qml:1148:21: QML StatusScrollView: Binding loop detected for property "contentWidth"
DBG 2022-11-14 11:53:44.989+03:00 NewBE_callPrivateRPC                       topics="rpc" tid=5576343 file=core.nim:20 rpc_method=stickers_installed
qrc:/app/AppLayouts/Chat/views/ChatMessagesView.qml:243:9: QML Connections: Cannot assign to non-existent property "onDataChanged"
DBG 2022-11-14 11:53:45.085+03:00 NewBE_callPrivateRPC                       topics="rpc" tid=5576343 file=core.nim:20 rpc_method=wakuext_getLatestVerificationRequestFrom
qrc:/app/AppLayouts/Onboarding/OnboardingLayout.qml:17: TypeError: Cannot read property 'displayBackButton' of null
qrc:/app/AppLayouts/Onboarding/stores/StartupStore.qml:7: TypeError: Cannot read property 'currentStartupState' of null
qrc:/app/AppLayouts/Onboarding/stores/StartupStore.qml:8: TypeError: Cannot read property 'selectedLoginAccount' of null
WRN 2022-11-14 11:53:45.135+03:00 Error decoding signal                      topics="signals-manager" tid=5576343 file=signals_manager.nim:43 err="Unknown signal received: mediaserver.started"
DBG 2022-11-14 11:53:45.146+03:00 NewBE_callPrivateRPC                       topics="rpc" tid=5576343 file=core.nim:20 rpc_method=settings_getSocialLinks
DBG 2022-11-14 11:53:45.147+03:00 NewBE_callPrivateRPC                       topics="rpc" tid=5576343 file=core.nim:20 rpc_method=settings_getSocialLinks
INF 2022-11-14 11:53:45.147+03:00 shutting down...                           topics="status-app" tid=5576343 file=nim_status_client.nim:145
Traceback (most recent call last)
/Users/anastasiya/status-desktop/src/nim_status_client.nim(181) nim_status_client
/Users/anastasiya/status-desktop/src/nim_status_client.nim(155) mainProc
/Users/anastasiya/status-desktop/src/app/boot/app_controller.nim(280) delete
/Users/anastasiya/status-desktop/src/app_service/service/message/service.nim(133) delete
/Users/anastasiya/status-desktop/vendor/nimqml/src/nimqml/private/qobject.nim(64) delete
SIGSEGV: Illegal storage access. (Attempt to read from nil?)
make: *** [run-macos] Error 1
```

--- a/src/app_service/common/types.nim
+++ b/src/app_service/common/types.nim
@@ -15,6 +15,7 @@ type
     Gap = 10
     Edit = 11
     DiscordMessage = 12
+    ContactIdentityVerification = 13
 
 type
   StatusType* {.pure.} = enum

--- a/src/app_service/service/activity_center/dto/notification.nim
+++ b/src/app_service/service/activity_center/dto/notification.nim
@@ -17,6 +17,7 @@ type ActivityCenterNotificationType* {.pure.}= enum
   CommunityRequest = 7
   CommunityMembershipRequest = 8
   CommunityKicked = 9
+  ContactVerification = 10
 
 type ActivityCenterMembershipStatus* {.pure.}= enum
   Idle = 0,

